examples/resnet_distributed_torch_app.py

```diff
@@ -9,7 +9,7 @@
 setup = 'echo \"alias python=python3\" >> ~/.bashrc && pip3 install --upgrade pip && \
     [ -d pytorch-distributed-resnet ] || \
     (git clone https://github.com/michaelzhiluo/pytorch-distributed-resnet && \
-    cd pytorch-distributed-resnet && pip3 install -r requirements.txt && \
+    cd pytorch-distributed-resnet && pip3 install -r requirements.txt torch==1.12.1+cu113 --extra-index-url https://download.pytorch.org/whl/cu113 && \
```

Comment:
(same Qs)

Q: why pin to 11.3, but not 11.6?

Q: does a 11.6/11.5 image support a torch cu113 wheel?

Reply:
ditto.